onnxruntime/contrib_ops/contrib_ops.cc

```diff
@@ -92,6 +92,37 @@ Sample echo operator.)DOC");
       .TypeAndShapeInferenceFunction(ONNX_NAMESPACE::propagateShapeAndTypeFromFirstInput)
       .SetDoc(R"DOC(Returns which elements of the input are NaN.)DOC");
 
+  ONNX_CONTRIB_OPERATOR_SCHEMA(Tokenizer)
```

Comment:
Should we mention about UTF-8 in the spec just to be super clear?

Reply:
Yes, will do. The author of the spec claimed that all strings in ONNX are UTF-8. Well, they use it for Bing. So there you go.

---
In reply to: [238894817](https://github.com/Microsoft/onnxruntime/pull/31#discussion_r238894817) [](ancestors = 238894817)